Remotion/ObjectBinding/Web/UI/Controls/BocRendererBase.cs

```diff
@@ -158,7 +158,16 @@ public virtual string CssClassReadOnly
     /// </remarks>
     public virtual string CssClassDisabled
     {
-      get { return "disabled"; }
+      get { return CssClassDefinition.Disabled; }
+    }
+
+    /// <summary>
+    /// Gets whether the control establishes a remotion theming context which automatically styles standard browser controls (e.g. input elements)
+    /// in the current theme. <c>false</c> by default. Derived classes may override this behavior. 
+    /// </summary>
+    public virtual bool EstablishesRemotionThemingContext
```

Comment:
I'm wondering if the instance members should just use `ThemingContext` and `CssClassThemd` instead of the `Remotion` prefix. Reasoning is that the const-value is predefined, but everything else just uses this in the more general concept of having a centrally themed element.